docs/en/cloud/manage/backups.md

```diff
@@ -101,3 +101,9 @@ You will need to reset the password for the new service in order to access it, y
 
 - Verify the data in the service
 - Delete the newly restored service once the data is verified
+
+## Undeleting or undropping tables
+
```

Comment:
One of the Engineers clarified that this command does not work on Cloud on account of replication however the other limit seems to be valid i.e. if the table/partition is > 1 TB